examples/next/src/components/CheckboxDemo.tsx

```diff
@@ -9,7 +9,7 @@ const CheckboxDemo = () => {
   const [isChecked, setIsChecked] = useState(true);
 
   return (
-    <Wrapper title="Checkbox">
+    <Wrapper title="Combobox">
```

Comment:
This file is named `CheckboxDemo`, but `Combobox` marked here

Reply:
Lmao, I updated the wrong file 😂